src/internal/packager2/parse/deprecated.go

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2021-Present The Zarf Authors
 
-package layout
+package parse
```

Comment:
Similar to parse.ZarfPackage, I think these can be private functions in `load` - maybe `load/parse.go`?